### ISSUE TITLE
Outdated bug report URL in man pages

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8600]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)| @scop reported|
|-|-|
|Trac ID|trac#8600|
|Type|defect|
|Created|2016-07-13 19:42:05Z|
|Branch|https://github.com/twisted/twisted/tree/8600-scop-bugsurl|

Bug report URLs in man pages point to http://twistedmatrix.com/bugs/ which gives a 404. Will add a PR once I can figure out what to write here so the trac spam checker will let this ticket through.

<details><summary>Searchable metadata</summary>

```
trac-id__8600 8600
type__defect defect
reporter__scop scop
priority__normal normal
milestone__None None
branch__8600_scop_bugsurl 8600-scop-bugsurl
branch_author__ 
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__None None
time__1468438925259096 1468438925259096
changetime__1468445154561801 1468445154561801
version__None None
owner__scop scop
cc__z3p
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)<a name="note_2"></a>|@scop commented|
|-|-|

https://github.com/twisted/twisted/pull/329

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @scop|
|-|-|

Thanks. Will merge!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_5"></a>|@adiroiban set status to `closed`|
|-|-|

In changeset 9a0d7b6b85011c897f757c91fbdfdfcc614726e8
```
#!CommitTicketReference repository="" revision="9a0d7b6b85011c897f757c91fbdfdfcc614726e8"
Merge 8600-scop-bugsurl: Fix url to bug reporting in man pages.

Author: scop
Reviewer: adiroiban
Fixes: #8600
```
